.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Issue informations
```

Comment:
I think that information doesn't take a 's' in english. it is already the plural. The singular is "a piece of information"

Reply:
Yes, it was typo mistake.